pandas/core/strings.py

```diff
@@ -978,6 +982,64 @@ def split(self, pat=None, n=-1, return_type='series'):
         result = str_split(self.series, pat, n=n, return_type=return_type)
         return self._wrap_result(result)
 
+    _shared_docs['str_partition'] = ("""
+    Split the string at the %(side)s occurrence of sep, and return a 3-tuple containing the part
+    before the separator, the separator itself, and the part after the separator.
+    If the separator is not found, return %(return)s.
+
+    Parameters
+    ----------
+    pat : string, default whitespace
+        String to split on.
+    return_type : {'series', 'frame'}, default 'frame'
+        If frame, returns a DataFrame (elements are strings)
```

Comment:
Small detail, but can you put a point `.` at the end of this sentence (as in html, the line breaks are not preserved, so the following `If series, ...` comes directly behind this, so without punctuation it looks strange